CMake/CircleCI/CircleCI_CTK_Docker.cmake

```diff
@@ -0,0 +1,89 @@
+##############################################################################
+#
+# Library:   CTK
+#
+# Copyright 2010 Kitware Inc. 28 Corporate Drive,
+# Clifton Park, NY, 12065, USA.
+#
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+##############################################################################
+
+
+set( CTK_SOURCE_DIR "/usr/src/CTK" )
+set( CTK_BINARY_DIR "/usr/src/CTK-build" )
+
+set( CTEST_SOURCE_DIRECTORY "${CTK_SOURCE_DIR}" )
+set( CTEST_BINARY_DIRECTORY "${CTK_BINARY_DIR}/CTK-build" )
+
+#############
+
+set( SITE_CTEST_MODE "Experimental" ) # Experimental, Continuous, or Nightly
+set( Experimental ON )
+
+set( SITE_CMAKE_GENERATOR "Unix Makefiles" ) # Ninja or Unix Makefiles
+set( CTEST_CMAKE_GENERATOR ${SITE_CMAKE_GENERATOR} )
+
+set( CTK_GIT_REPOSITORY "https://github.com/commontk/CTK.git" )
+
+set( CTEST_CTEST_COMMAND "ctest" )
+
+set( CTEST_COMMAND "ctest" )
```

Comment:
These shouldn't be needed. Invoking the script with ctest should automatically set these variables.